pkg/knode-manager/adapters/k8s/pod.go

```diff
@@ -441,7 +419,7 @@ func (p *PodAdapter) createServiceAccount(ctx context.Context, secret *corev1.Se
 }
 
 func (p *PodAdapter) Create(ctx context.Context, pod *corev1.Pod) error {
-	if pod.Namespace == "kube-system" {
+	if pod.Namespace == SkipNamespace {
```

Comment:
what does Skipnamespace mean？
I prefer reserverdNS

Reply:
ok, i will change `Skipnamespace` to `ReserverdNS`

Reply:
I may make a mistake
`reserved` not `reserverd`

Reply:
updated